util/mailer.py

```diff
@@ -0,0 +1,32 @@
+import smtplib
+import ssl
+
+from flask import current_app as app
+
+
+def send_mail(receiver, message):
+    """Sends a new mail to the given receiver.
+    :param receiver: The receiver's mail address.
```

Comment:
Please use the same syntax as before: ":param str receiver: The receiver's mail address."